### PR TITLE
Gutenboarding: update templates

### PR DESCRIPTION
**Pull request by johnHackworth**
Not really testable yet (until we merge D41281-code, at least).

This PR updates the templates used on Gutenboarding. It also changes the names of the files to correspond with the templates and adds a new one that was still not there.

We'll probably need to merge this right away by now, but I'll do it at some other time that it's not super late at night and I can realize if something is wrong :)

### CODE REVIEW

**Comment by simison** [comment]
Doyle template (`static/page-templates/templates/doyle.json`) still seems to have private images so that needs fixing and re-importing.

E.g. this file:
```
https://gutenbaordinghp.files.wordpress.com/2020/03/christian-lunde-gf8xg0a0xdy-unsplash.jpg?w=683
```

@iamtakashi @ianstewart @alaczek  can you have a look?

File: static/page-templates/templates/vesta.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (simison) [inline static/page-templates/templates/vesta.json:1]: @iamtakashi  this template has links to `https://iamtakashinuxtest20200316221813.wordpress.com/wp-admin/post.php?post=33&amp;action=edit#` — is that expected?

Images are also referenced from `iamtakashinuxtest20200316221813.files.wordpress.com` — is that expected as well?

**Comment by simison** [comment]
Something is broken with this Camden template @johnHackworth — maybe wrong tempalte reference in preview URL?

image

**Comment by simison** [comment]
Something funky with Vesta:

`https://public-api.wordpress.com/rest/v1/template/demo/mayland/vesta/?font_base=Raleway&font_headings=Cabin&site_title=Vesta`

image

File: static/page-templates/templates/vesta.json (hunk 1 above)
Comment (iamtakashi) [reply static/page-templates/templates/vesta.json:1]: I believe the content including images still need to be polished and optimised, and I don't think we've decided where to save the markup and the images. cc @ianstewart

File: client/landing/gutenboarding/available-designs.ts [hunk 2]
```diff
@@ -39,7 +39,7 @@ const availableDesigns: Readonly< AvailableDesigns > = {
 		{
 			title: 'Camden',
 			slug: 'Camden',
-			template: 'maywood2',
+			template: 'camdem',
```
Comment (simison) [inline client/landing/gutenboarding/available-designs.ts:42]: _Camden_ or _Camdem_?

Comment (simison) [reply client/landing/gutenboarding/available-designs.ts:42]: Fix https://github.com/Automattic/wp-calypso/pull/40799

**Comment by simison** [comment]
>Something is broken with this Camden template

Fix: https://github.com/Automattic/wp-calypso/pull/40799

**Comment by sirreal** [comment]
> Something funky with Vesta:
> 
> `https://public-api.wordpress.com/rest/v1/template/demo/mayland/vesta/?font_base=Raleway&font_headings=Cabin&site_title=Vesta`
> 
> image

With D41466-code and #40887 we can debug this.

**Pull request merged**